module/VuFindSearch/src/VuFindSearch/Backend/Blender/Response/Json/RecordCollection.php

```diff
@@ -374,6 +374,28 @@ protected function mapFacetValues(array $collections, array $settings): array
                 }
             }
         }
+
+        foreach ($settings['Mappings'] as $backendId => $mappings) {
+            if (($collections[$backendId] ?? false) &&
+                ($ignore = $mappings['Ignore'] ?? '') &&
+                ($mappings['CountIgnored'] ?? true))
+            {
+                if (is_array($ignore)) {
```

Comment:
You could simplify this code significantly by reducing the big if..else down to:

```
foreach (is_array($ignore) ? $ignore : $result as $ignoredValue) {
   ...
}
```

Reply:
Yes, your code looks a lot better. I just don't know how to simplify the outer if clause.